.zuul.yaml

```diff
@@ -5,6 +5,7 @@
         - cloud-provider-openstack-acceptance-test-keystone-authentication-authorization
         - cloud-provider-openstack-acceptance-test-lb-octavia
         - cloud-provider-openstack-acceptance-test-e2e-conformance
+        - cloud-provider-openstack-acceptance-test-e2e-conformance-latest-release
```

Comment:
can we rename `latest-release` to `stable-branch-1.10` please?

Reply:
That means we have to update this yaml file every time when we release a new branch ?

Reply:
@dims , of course, `stable-branch-1.10` is more exact, thank you.

Reply:
@dixudx  over time we will accumulate 3 of these (just like k8s supports 3 last branches) and we will have to drop the oldest one when we go beyond 3

Reply:
@dixudx yes, if we use explicit version job name, we have to update .zuul.yaml when k8s release; if we use current name, we only need to update OpenLab zuul job define.